src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const rb1 = Object.keys(robot);
+  const rb2 = Object.values(robot);
+  const obj = {};
+
+  for (let i = 0; i < rb1.length; i++) {
+    for (let g = i + 1; g < rb1.length; g++) {
+      if (rb1[i] === rb1[g] || rb2[i] === rb2[g]) {
+        return null;
+      }
+    }
```

Comment:
You definitely have to avoid nested loops in your code

Reply:
Btw your variable names are unclear. You should think of sth that helps to understand what is stored inside the variable